index.js

```diff
@@ -29,14 +27,14 @@ function main(fn, fp, data, opts) {
 		});
 	}
 
-	var json = JSON.stringify(data, opts.replacer, opts.indent) + '\n';
+	const json = JSON.stringify(data, opts.replacer, opts.indent) + '\n';
 
 	return fn(fp, json, {mode: opts.mode});
 }
 
 module.exports = function (fp, data, opts) {
```

Comment:
That's incorrect. ES2015 has a whole section on function name inference.

http://www.2ality.com/2015/09/function-names-es6.html